packages/identity-service/src/routes/idSignals.js

```diff
@@ -28,10 +28,6 @@ module.exports = function (app) {
       const [
```

Comment:
only AAO uses this, and it will get the social handles from DN in the future. it will also use identity as fallback to get socials of users who updated their socials while on old clients.